src/bundle/Resources/views/themes/admin/ui/error_page/403.html.twig

```diff
@@ -1,4 +1,4 @@
-{% extends '@ezdesign/ui/layout.html.twig' %}
+{% extends '@ezdesign/ui/layout_error.html.twig' %}
 
 {% block navigation %}{% endblock %}
```

Comment:
if this block does not exist in layout_error you probably can remove it from those templates